build.properties

```diff
@@ -90,9 +90,9 @@ javadoc.top=<div style="padding: 6px; text-align: center; font-size: 80%;">This
 jdk.docs.version=23
 jfx.build.jdk.version=23
 jfx.build.jdk.buildnum=37
-jfx.build.jdk.version.min=22
-jfx.build.jdk.buildnum.min=36
-jfx.jdk.target.version=22
+jfx.build.jdk.version.min=23
+jfx.build.jdk.buildnum.min=37
+jfx.jdk.target.version=23
```

Comment:
gradle 8.5 does not support JDK23 so jfx.gradle.version.min should be updated to 8.10.2
ref: https://docs.gradle.org/8.10.2/release-notes.html